src/services/WolfDen.Domain/Entity/LeaveRequest.cs

```diff
@@ -0,0 +1,40 @@
+﻿using WolfDen.Domain.Enum;
+
+namespace WolfDen.Domain.Entity
+{
+    public class LeaveRequest
+    {
+        public int Id { get; private set; }
+        public int EmployeeId { get; private set; }
+        public virtual Employee Employee { get; private set; }
```

Comment:
you can move navigation properties to last - for better readability

Reply:
Navigation Property Moved to Last of Model Class